src/isar_exr/api/energy_robotics_api.py

```diff
@@ -66,3 +66,46 @@ def pause_current_mission(self, exr_robot_id: str) -> None:
         ]
         if not success:
             raise RobotException(f"Invalid status after pausing mission: '{status}'")
+
+    def create_mission_definition(
+        self, site_id: str, mission_name: str, exr_robot_id: str
+    ) -> str:
+        mutation_string: str = """
+            mutation createMissionDefinition($site_id:String!, $mission_name:String!, $exr_robot_id:CreateRequiredRobotConfigInput) {
+                createMissionDefinition(
+                input: { siteId: $site_id, name: $mission_name, requiredRobotConfig: $exr_robot_id }
+                ) {
+                id
+            }}
+        """
+        params: dict = {
+            "site_id": site_id,
+            "mission_name": mission_name,
+            "exr_robot_id": exr_robot_id,
+        }
+
+        try:
+            response_dict: dict[str, Any] = self.client.query(mutation_string, params)
+        except Exception as e:
+            raise RobotException(e)
+
+        mission_id = response_dict["createMissionDefinition"]["id"]
+        return mission_id
+
+    def star_mission_exicution(self, mission_id: str, robot_id: str) -> None:
+        mutation_string: str = """
+            mutation startMissionExecution($robot_id:ID!, $mission_id:String!) {
+                startMissionExecution(
+                    input: { robotID: $robot_id, missionDefinitionID: $mission_id }
+                ) {
+                    id
+                }
+            }
+        """
+        params: dict = {"robot_id": robot_id, "mission_id": mission_id}
+
+        try:
+            response_dict: dict[str, Any] = self.client.query(mutation_string, params)
+        except Exception as e:
+            raise RobotException(e)
+        mission_id = response_dict["startMissionExecution"]["id"]
```

Comment:
should this be stored somewhere or returned?

Reply:
sure, done